src/charm.py

```diff
@@ -21,6 +21,10 @@
 from services.s3 import S3BucketWrapper, validate_s3_bucket_name
 
 SECRETS_FILES = ["src/secrets/seldon_secret.yaml.j2", "src/secrets/pipelines_secret.yaml.j2"]
+PODDEFAULTS_FILES = [
+    "src/poddefaults/poddefault-example-minio.yaml.j2",
+    "src/poddefaults/poddefault-example-mlflow.yaml.j2",
+]
```

Comment:
can we rename these to not say "example"?  When I read "example" I think they're some hard-coded structure, but really they're templates.  Also, if you're putting them in a poddefaults dir, they don't need the poddefault suffix.

Maybe:
```suggestion
PODDEFAULTS_FILES = [
    "src/templates/poddefault-minio.yaml.j2",
    "src/templates/poddefault-mlflow.yaml.j2",
]```